conftest.py

```diff
@@ -55,3 +57,9 @@ def monkey_allclose(a, b, rtol=1.e-5, atol=1.e-8):
         return real_all_close(a, b, rtol=rtol, atol=atol)
 
     monkeypatch.setattr(np.testing, 'assert_allclose', monkey_allclose)
+
+
+@pytest.fixture(autouse=True)
+def reset_rng():
```

Comment:
Please keep in mind that hypothesis also tries to manage `{np.,}random` state. So the seed here might not actually be what's used in a test, depending on fixture exec order.